tests/Microsoft.Identity.Web.Test/TestConstants.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Identity.Web.Test
+{
+    internal static class TestConstants
+    {
+        public const string ProductionPrefNetworkEnvironment = "login.microsoftonline.com";
+        public const string ProductionPrefNetworkUSEnvironment = "login.microsoftonline.us";
+        public const string ProductionNotPrefEnvironmentAlias = "sts.windows.net";
+
+        public const string LocalHost = "https://localhost";
+        
+        public const string ApiAudience = "api://1EE5A092-0DFD-42B6-88E5-C517C0141321";
```

Comment:
Wouldn't you want to have "api://" + ApiClientId  ?

Reply:
yes. thanks. will fix.

---
In reply to: [381455089](https://github.com/AzureAD/microsoft-identity-web/pull/22#discussion_r381455089) [](ancestors = 381455089)